src/memfs.ts

```diff
@@ -196,6 +196,13 @@ export class InMemoryFileSystem extends EventEmitter
     this.overlay.set(uri, text);
   }
 
+  /**
+  * Called by TS service to scan virtual directory when TS service looks for source files that belong to a project
+  */
+  public readDirectory(rootDir: string, extensions: string[], excludes: string[], includes: string[]): string[] {
+      return [];
+  }
```

Comment:
Why do we need this method?

Reply:
Class `InMemoryFileSystem` must implement interface `ParseConfigHost` and that method was missing.

Reply:
I don't think we need to implement either of `ts.ParseConfigHost, ts.ModuleResolutionHost`.
I think we can remove this function for now (as well as the `implements ts.ParseConfigHost, ts.ModuleResolutionHost`)